sormas-ui/src/test/java/de/symeda/sormas/ui/TestDataCreator.java

```diff
@@ -253,7 +253,8 @@ public PathogenTestDto createPathogenTest(SampleReferenceDto sample, PathogenTes
 		PathogenTestDto sampleTest = PathogenTestDto.build(sample, labUser);
 		sampleTest.setTestType(testType);
 		sampleTest.setTestDateTime(testDateTime);
-		sampleTest.setLab(lab);
+		sampleTest.
+				setLab(lab);
```

Comment:
How did this line break happen? :D

Reply:
was testing if you catch me :)
joking.. i have no clue :D , reverted it